File: krylan-swift/Sources/Theme.swift
// Бренд и палитра KRYLAN.
import SwiftUI

enum Brand {
    static let name   = "KRYLAN"
    static let slogan = "Дай устройству крылья"
    static let author = "Кырлан Александр Сергеевич"
    static let version = "0.6.0"

    // Палитра navy «рубка» KRYLAN (эталон HUD macOS CleanMac).
    static let bg0    = Color(red: 0.035, green: 0.075, blue: 0.153)   // #091327
    static let bg1    = Color(red: 0.024, green: 0.051, blue: 0.118)   // #060d1e — край свечения
    static let glow   = Color(red: 0.090, green: 0.239, blue: 0.447)   // #173d72 — центр свечения
    static let glass  = Color(red: 0.063, green: 0.141, blue: 0.267)   // #102444
    static let track  = Color(red: 0.133, green: 0.263, blue: 0.435)   // #22436f
    static let text   = Color(red: 0.894, green: 0.933, blue: 0.984)   // #e4eefb
    static let muted  = Color(red: 0.439, green: 0.533, blue: 0.698)   // #7088b2
    static let green  = Color(red: 0.184, green: 0.898, blue: 0.627)   // #2fe5a0
    static let blue   = Color(red: 0.184, green: 0.561, blue: 1.0)     // #2f8fff
    static let cyan   = Color(red: 0.212, green: 0.839, blue: 1.0)     // #36d6ff
    static let purple = Color(red: 0.749, green: 0.353, blue: 0.949)   // #BF5AF2 (запас)
    static let yellow = Color(red: 1.0,   green: 0.839, blue: 0.039)   // #ffd60a
    static let red    = Color(red: 1.0,   green: 0.353, blue: 0.322)   // #ff5a52

    /// Цвет по нагрузке: 0 — хорошо (зелёный), 100 — плохо (красный).
    static func load(_ p: Double) -> Color {
        p < 60 ? green : (p < 85 ? yellow : red)
    }
}

/// Заголовок страницы: на iOS его показывает нав-бар, поэтому рендерим только на macOS.
struct PageHeader: View {
    let title: String
    var body: some View {
        #if os(macOS)
        Text(title).font(.largeTitle.bold()).foregroundStyle(Brand.text)
        #else
        EmptyView()
        #endif
    }
}

File: krylan-android/app/src/main/java/com/krylan/app/ui/Theme.kt
// Бренд и палитра KRYLAN для Android.
package com.krylan.app.ui

import androidx.compose.foundation.isSystemInDarkTheme
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.darkColorScheme
import androidx.compose.runtime.Composable
import androidx.compose.ui.graphics.Color

object Brand {
    const val NAME = "KRYLAN"
    const val SLOGAN = "Дай устройству крылья"
    const val AUTHOR = "Кырлан Александр Сергеевич"

    // HUD-«рубка» CleanMac: navy-база, glass-карточки, неон-акценты.
    val bg0 = Color(0xFF091327)     // navy фон
    val glowIn = Color(0xFF173D72)  // центр радиального свечения
    val glowOut = Color(0xFF060D1E) // края радиального свечения
    val glass = Color(0xFF102444)   // стекло
    val track = Color(0xFF22436F)   // дорожка гейджа
    val text = Color(0xFFE4EEFB)
    val muted = Color(0xFF7088B2)
    val green = Color(0xFF2FE5A0)
    val blue = Color(0xFF2F8FFF)
    val yellow = Color(0xFFFFD60A)
    val red = Color(0xFFFF5A52)
    val purple = Color(0xFFBF5AF2)  // оставлен для совместимости (карточки)
    val cyan = Color(0xFF36D6FF)

    fun load(p: Float): Color = if (p < 60) green else if (p < 85) yellow else red
}

private val KrylanColors = darkColorScheme(
    primary = Brand.green,
    background = Brand.bg0,
    surface = Brand.glass,
    onBackground = Brand.text,
    onSurface = Brand.text,
)

@Composable
fun KrylanTheme(content: @Composable () -> Unit) {
    MaterialTheme(colorScheme = KrylanColors, content = content)
}
